src/pages/SignUpPage.js

```diff
@@ -18,15 +21,15 @@ const SignUpPage = () => {
   }
 
   return (
-    <div className="container-fluid h-100 signUpContainer">
+    <AuthWrapper>
       <p>
         <FormattedMessage id="signup.title" />
       </p>
       <SignUpForm onSubmit={signUpRequest} />
-      <Link to={routes.login}>
+      <Link className="signInLink" to={routes.login}>
```

Comment:
Naming conventions in CSS state that class names should stick to the kebab case convention. This would look like: 
`signin-link`. 

You can read more about it [here](https://cssguidelin.es/#naming-conventions)

Reply:
Thanks!